docs/language/operators/README.md

```diff
@@ -685,8 +685,8 @@ time                        uid
 
 |                           |                                                 |
 | ------------------------- | ----------------------------------------------- |
-| **Description**           | Add/update fields based on the results of an expression         |
-| **Syntax**                | `put <field> := <expression> [, <field> := <expression> ...]`   |
+| **Description**           | Add/update fields based on the results of an expression<br><br>The `put` keyword is optional, though its explicit use may be appropriate to disambiguate from where `:=` is used for [naming fields to hold aggregation results](../aggregate-functions/README.md#field-naming). |
```

Comment:
@mattnibs @nwt @mccanne: Please keep me honest here! I _think_ I've listed the only other context in the language where `:=` comes up, but I sometimes have difficulty keeping it all mentally cached.

Reply:
No you do not need to use `:=`. For implied puts the semantic compiler is able to figure out if a call is a normal function (i.e. a call in a put) or an aggregation call (i.e. part of summarize proc).